core/src/test/resources/nelson/manifest.v1.everything.yml

```diff
@@ -108,6 +108,11 @@ plans:
       - ref: s3
         uri: http://s3.aws.com
     expiration_policy: retain-latest-two-major
+    volumes:
+      - name: an-empty-dir
+        mountPath: /foo/bar
+        emptyDir:
```

Comment:
I'm not sure we should be specific about the kind of volume here based on our discussion at lunch. If we go for templating later, then we can simply have the user specify the size of the volume they'd like. WDYT?